tests/http_spec.rs

```diff
@@ -95,6 +95,9 @@ struct HttpSpec {
   #[serde(default)]
   mock: Vec<Mock>,
 
+  #[serde(default)]
+  env: BTreeMap<String, String>,
```

Comment:
why BTreeMap and not HashMap as it is specified in server_context?

Reply:
changed it